zeroex/ordervalidator/order_validator_test.go

```diff
@@ -210,6 +210,131 @@ func TestBatchValidateAValidOrder(t *testing.T) {
 	assert.Equal(t, orderHash, validationResults.Accepted[0].OrderHash)
 }
 
+func TestBatchOffchainValidateUnsupportedStaticCall(t *testing.T) {
+	if !serialTestsEnabled {
+		t.Skip("Serial tests (tests which cannot run in parallel) are disabled. You can enable them with the --serial flag")
+	}
+
+	teardownSubTest := setupSubTest(t)
+	defer teardownSubTest(t)
+
+	ethClient := ethclient.NewClient(rpcClient)
+	signedOrder := scenario.CreateZRXForWETHSignedTestOrder(t, ethClient, makerAddress, takerAddress, wethAmount, zrxAmount)
+	// NOTE(jalextowle): This asset data encodes a staticcall to a function called `unsupportedStaticCall`
+	testSignedOrder := signedOrderWithCustomMakerFeeAssetData(t, *signedOrder, common.Hex2Bytes("c339d10a000000000000000000000000692a70d2e424a56d2c6c27aa97d1a86395877b3a0000000000000000000000000000000000000000000000000000000000000060c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a47000000000000000000000000000000000000000000000000000000000000000048b24020700000000000000000000000000000000000000000000000000000000"))
+	signedOrders := []*zeroex.SignedOrder{
+		&testSignedOrder,
+	}
+
+	rateLimiter := ratelimit.NewUnlimited()
+	rpcClient, err := rpc.Dial(constants.GanacheEndpoint)
+	require.NoError(t, err)
+	ethRPCClient, err := ethrpcclient.New(rpcClient, defaultEthRPCTimeout, rateLimiter)
+	require.NoError(t, err)
+
+	orderValidator, err := New(ethRPCClient, constants.TestChainID, constants.TestMaxContentLength, ganacheAddresses)
+	require.NoError(t, err)
+
+	accepted, rejected := orderValidator.BatchOffchainValidation(signedOrders)
+	assert.Len(t, accepted, 0)
+	require.Len(t, rejected, 1)
+	testSignedOrder.ResetHash()
+	expectedOrderHash, err := testSignedOrder.ComputeOrderHash()
+	require.NoError(t, err)
+	assert.Equal(t, expectedOrderHash, rejected[0].OrderHash)
```

Comment:
As I mentioned in Slack, let's actually check the `RejectedOrderStatus` here instead of just checking that the order was rejected.